prow/scripts/cluster-integration/compass-gke-benchmark.sh

```diff
@@ -139,7 +139,7 @@ function installHelm() {
 }
 
 function installKyma() {
-  KYMA_CLI_VERSION="2.1.3"
+  KYMA_CLI_VERSION="2.2.0"
```

Comment:
Why do you use so old version of Kyma CLI? Kyma CLI 2.9.1 was already released.

Reply:
That's the current version we are adopting in Compass. We have regular tasks to adopt new Kyma versions but it's not a trivial process.